db/schema.rb

```diff
@@ -16,24 +16,24 @@
 
   create_table "allyships", force: :cascade do |t|
     t.integer "user_id"
-    t.datetime "created_at", precision: nil
```

Comment:
Can you remove the changes to this file from your PR? It's not related to your changes and we don't need to update the schema.